### DISCUSSION TITLE
p/talk-about-chat-gpt/

### DISCUSSION THREAD

**Discussion by giscus** [debuginn/blog#245] [General]
# p/talk-about-chat-gpt/

前言 说起我真正使用落地的产品是从 22 年 11 月份开始，那时候 ChatGPT 刚发布灰度出来，面对这个对话框一样的界面，我发现这个简简单单甚至有些简陋的页面，竟然是技术变革的入口，看过大模型相关理论以及原理的同学，大家会发现，这个 AI 已经学习了我们近十年的信息，也就是说，他的知识储备已经高于我们在这个世界上的任何一个人，同一时间段，已经看到 V2EX 已经有人开始将高考试题让 GPT 去做了，当时的 GPT 可以考上一所不错的一本院校了，现在的 GPT4 我想应该是在全领域领先于各个行业了。
不出意外，ChatGPT 在全球理所当然的火了起来，他带来的是一场技术的革命，他区别于元宇宙、VR 或者区块链，因为元宇宙、VR 技术的发展到现在，需要的是基础材料的突破，当时现在（23年10月）还没有让人振奋人心的消息传来，区块链延伸而来的比特币，成为了全球热爱博彩的圣地、也成为了进行洗钱的工具，对社会和全球的发展没有进步意义，同时又白白消耗了大量的电力资源。 GPT 是利用现有的计算机科学技术进行突破，商业化速度比预想的要快很多，微软的 Edge 浏览器率先融合进了 GPT，成为了第一款商业化的产品。
工具 从接触使用 GPT 开始，发现自己逐步离不开这个工具，从基本的文章润色到代码的讲解及优化建议、再到提需求做一个小的工具，他的回答都令我非常满意，目前在工作生活中，都在使用了大量的 GPT 衍生工具，在这里也分享给大家：
ChatGPT 使用最多的就是自家的网站与 APP，也就是我文章开头所说的 “对话框界面”。
OpenAI 官方网址：https://openai.com/
ChatGPT 官方网址：https://chat.openai.com/
MacGPT 由于现在官方没有发布 Mac 的桌面 ChatGPT APP，这款是目前感觉使用体验最好的软件了，可以在顶部工具栏和侧边栏呼出，使用起来非常便捷。
Chat2DB 这是一款数据库连接工具，通过 ChatGPT 或者其他的 GPT 工具将自然语言处理成 SQL 语言，也可以基于数据生成报表，使用起来十分便捷与方便。
Github 地址：https://github.com/chat2db/Chat2DB/
感悟 从人类进入了刀耕火种的时代开始，大家就已经学会了使用工具，现在的 GPT 同样也是一种工具，目前，他会一定程度的帮助我们在工作中提升效率，有一些重复性的工作甚至会被他代替，事实也是如此。
目前我们还是将 GPT 的强大功能简单的接入到我们日常使用的工具中，这是最初的阶段，未来，GPT 会衍生出每个人的电子助理，甚至在未来，就如流浪地球的 MOSS 机器人一样，他们将有独立的意识。
不过在使用目前的产品中，发现了一些问题，就是他不知道他不知道，对于一些常识或者一些技能问题的回答，深度使用的话，不难发现，他就是在用关键参数或者关键词在创造答案，这个答案的可信度并不是那么高，甚至就是错误的。
前两天看到博客中有聊到 《ChatGPT 导致了 Stack Overflow 访问量的下降？ 》 大家遇到问题不再集中思考来复现问题，发现更深层次的问题，而是通过 GPT 来快问快答，GPT3.

https://blog.debuginn.com/p/talk-about-chat-gpt/

**Comment by kuang2714** [comment]
博主你好，茶备案项目新开张，希望得到更大的支持加入茶备案！注册得到您的专属网站备案号！希望博主多多支持！https://icp.redcha.cn
希望博主能在我们网站申请一个备案号放在页脚呀！感谢博主的支持
如果被打扰了，那就万分抱歉

**Reply by debuginn** [reply]
不好意是哈 不感兴趣 不过为你的项目点赞
